package.json

```diff
@@ -173,7 +173,8 @@
     "prepare": "yarn contracts:compile && yarn i18n:compile",
     "prepublishOnly": "yarn widgets:build",
     "test:e2e": "start-server-and-test 'serve build -l 3000' http://localhost:3000 'cypress run --record'",
-    "widgets:start": "cosmos",
+    "cosmos:run": "cosmos",
+    "cosmos:export": "NODE_ENV=development cosmos-export",
```

Comment:
Setting the env to `development` is necessary otherwise one Webpack plugin complains about being in a production env and I couldn't get past that